### PR TITLE
Minor improvements for German translation

### PR DESCRIPTION
**Pull request by thisdudeisvegan**
### First time contributor checklist
- [X] I have read the [README](https://github.com/WhisperSystems/Signal-iOS/blob/master/README.md) and 
- [X] I have signed the [Contributor Licence Agreement](https://whispersystems.org/cla/)

### Contributor checklist
- [X] I'm following the [code, UI and style conventions](https://github.com/WhisperSystems/Signal-iOS/blob/master/CONTRIBUTING.md#code-conventions)
- [X] My commits are rebased on the latest master branch
- [X] My commits are in nice logical chunks
- [X] My contribution is fully baked and is ready to be merged as is
- [X] I have tested my contribution on these devices:
 * iPhone 12 Pro Max, iOS 14.2

- - - - - - - - - -

### CODE REVIEW

File: Signal/translations/de.lproj/Localizable.strings [hunk 1]
```diff
@@ -230,10 +230,10 @@
 "ATTACHMENT_ERROR_INVALID_FILE_FORMAT" = "Anhang besitzt ein ungültiges Dateiformat.";
 
 /* Attachment error message for attachments without any data */
-"ATTACHMENT_ERROR_MISSING_DATA" = "Anhang ist leer.";
+"ATTACHMENT_ERROR_MISSING_DATA" = "Der Anhang ist leer.";
 
 /* Accessibility hint describing what you can do with the attachment button */
-"ATTACHMENT_HINT" = "Zu sendende Medieninhalte auswählen";
+"ATTACHMENT_HINT" = "Medieninhalte zum versenden auswählen";
```
Comment (TristanBandat) [inline Signal/translations/de.lproj/Localizable.strings:236]: "Medieninhalte zum Versenden auswählen" would be correct. Words after 'zum' are starting almost always with a uppercase letter!

**Review by TristanBandat — CHANGES_REQUESTED** [review]
Found a small typo at line 236. Please change, thanks!

**Comment by ZER0-X** [comment]
There is a special platform for translations, modification is not done directly to the code.

**Comment by Plarome** [comment]
Translation is done here: https://www.transifex.com/signalapp/public/

**Pull request closed without merging**